pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1083,8 +1083,9 @@ type ReplicatedSpec struct {
 	Size uint `json:"size"`
 
 	// TargetSizeRatio gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity
+	// +kubebuilder:validation:Minimum=0
 	// +optional
-	TargetSizeRatio float64 `json:"targetSizeRatio,omitempty"`
+	TargetSizeRatio *float64 `json:"targetSizeRatio,omitempty"`
```

Comment:
Actuall let's not change to a pointer type, I misunderstood your earlier question. Let's leave the type unchanged to ensure backward compatibility.